src/platform/user/profile/vap-svc/components/base/VAPServiceEditModalErrorMessage.jsx

```diff
@@ -68,21 +63,20 @@ export default function VAPServiceEditModalErrorMessage({
     default:
       content = (
         <p>
-          We’re sorry. We can’t save your {title.toLowerCase()} at this time.
-          We’re working to fix this problem. Please try again or check back
-          soon.
+          We're sorry. We can't update your information right now. We're working
+          to fix this problem. Please check back later.
         </p>
       );
   }
 
   return (
-    <AlertBox
-      className="vads-u-margin-top--0"
-      content={<div className="columns">{content}</div>}
-      isVisible
-      onCloseAlert={clearErrors}
-      scrollOnShow
-      status={ALERT_TYPE.ERROR}
-    />
+    <div
```

Comment:
🤔 we should be able to accomplish this with the `AlertBox` component using the `background-only` prop

Reply:
Yeah, I thought about that. But then a realized that I would have to override the icon and the background color.

Reply:
you should get the background color. you just need to do some custom content to handle the icon, but you are doing that part anyway.

Reply:
I just realized that the "success" alert is probably the best thing to base this error alert on https://github.com/department-of-veterans-affairs/vets-website/blob/a7c97a64abdec88203b7b820446cb93cbb921a22/src/platform/user/profile/vap-svc/components/ContactInformationFieldInfo/ContactInformationUpdateSuccessAlert.jsx#L40-L50

Reply:
I didn't know AlertBox would accept `children`.

Reply:
@erikphansen give it a try now.